packages/core/core/src/requests/TargetRequest.js

```diff
@@ -135,64 +158,80 @@ export class TargetResolver {
           return matchingTarget;
         });
       } else {
+        optionTargets;
```

Comment:
That can be removed

Reply:
ya, not sure how that ended up in there. removed.